.github/workflows/ci.yml

```
@@ -166,6 +180,7 @@ jobs:
 
       - name: "Cargo test"
         run: |
+          export UV_INDEX_URL="http://localhost:80/simple"
```

```suggestion
          export UV_INDEX_URL="http://localhost/simple"
```

Shouldn't be a need to specify the port in this case